api/api_ethereum.go

```diff
@@ -1098,25 +1096,29 @@ func (args *EthTransactionArgs) setDefaults(ctx context.Context, b Backend) erro
 	if args.MaxPriorityFeePerGas == nil || args.MaxFeePerGas == nil {
 		if b.ChainConfig().IsEthTxTypeForkEnabled(head.Number) && args.GasPrice == nil {
 			if args.MaxPriorityFeePerGas == nil {
-				// TODO-Klaytn: Original logic of Ethereum uses b.SuggestTipCap which suggests TipCap, not a GasPrice.
-				// But Klaytn currently uses fixed unit price determined by Governance, so using b.SuggestPrice
-				// is fine as now.
-				args.MaxPriorityFeePerGas = (*hexutil.Big)(fixedGasPrice)
+				args.MaxPriorityFeePerGas = (*hexutil.Big)(gasPrice)
 			}
 			if args.MaxFeePerGas == nil {
 				// TODO-Klaytn: Calculating formula of gasFeeCap is same with Ethereum except for
 				// using fixedBaseFee which means gasFeeCap is always same with args.MaxPriorityFeePerGas as now.
+				baseFee := fixedBaseFee
+				if isKIP71 {
+					baseFee = gasPrice
+				}
 				gasFeeCap := new(big.Int).Add(
 					(*big.Int)(args.MaxPriorityFeePerGas),
```

Comment:
Do we have to add this to maxFeePerGas??? 
I think our recommendation is `baseFee * 2` for maxFeePerGas.
If i misunderstand, please let me know!

Reply:
yes, right. 
`baseFee*2` is computed at https://github.com/bobpkr/klaytn/blob/api-dynamicGasPrice/api/api_ethereum.go#L1108-L1111

Reply:
Yes, but what is the recommended value for `maxFeePerGas`?
`baseFee*2 + maxPriorityFeePerGas`? or `baseFee * 2`?

Reply:
I misunderstood. I will check it out and fix it as you suggested. thanks.

Reply:
How about changing like below?

```
// After KIP-71 hard fork, `gasFeeCap` was set to `baseFee*2` by default.
gasFeeCap := new(big.Int).Mul(gasPrice, big.NewInt(2))
if !isKIP71 {
    // Before KIP-71 hard fork, `gasFeeCap` was set to `baseFee*2 + maxPriorityFeePerGas` by default.
    gasFeeCap = new(big.Int).Add(gasFeeCap, (*big.Int)(args.MaxPriorityFeePerGas))
}
```